### PR TITLE
chore: remove debugging

### PR DESCRIPTION
**Pull request by crisog**
After a few console.log() additions to Pocket script to fix a bug, this PR removes them as the issue was fixed.

### CODE REVIEW

**Review by adamsoffer — APPROVED** [review]
LGTM

File: cmd/pocket.ts [hunk 1]
```diff
@@ -76,14 +76,9 @@ const pocketImport = async () => {
 
   const aggsLimitDate = new Date("2021-09-17");
 
-  console.log(`Date from: ${fromDate}`);
-  console.log(`Date to: ${toDate}`);
-  console.log(`List of days: ${days}`);
-  console.log(`Days difference: ${dateDiff}`);
-
   for (const day of days) {
+    let timeUnitMsg = "on day";
```
Comment (crisog) [inline cmd/pocket.ts:80]: should've declared this out of the loop but its ok

Comment (crisog) [reply cmd/pocket.ts:80]: noted for next PR

Comment (adamsoffer) [reply cmd/pocket.ts:80]: oh yeah. I can take that one

**Pull request merged**